fem/fespace.hpp

```diff
@@ -169,31 +187,80 @@ class FiniteElementSpace
        boundary. */
    void BuildNURBSFaceToDofTable() const;
 
+   typedef std::uint64_t VarOrderBits;
+   enum { MaxVarOrder = 8*sizeof(VarOrderBits) - 1 };
```

Comment:
Rather than an `enum` maybe this could be a `static constexpr`?

Reply:
OK